recipes-openxt/tss/tpm2-tools_git.bb

```diff
@@ -4,21 +4,17 @@ SECTION = "tpm"
 
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=91b7c548d73ea16537799e8060cea819"
-DEPENDS = "tpm2-tss openssl curl autoconf-archive pkgconfig"
-SRC_URI = "git://github.com/01org/tpm2-tools.git;protocol=git;branch=master;name=tpm2-tools;destsuffix=tpm2-tools \
-    file://0001-tpm2_listpcrs-use-TPM2_GetCapability-to-determine-PC.patch \
-    file://0002-listpcrs-remove-one-redundant-call-to-tpm-get-cap.patch \
-    file://0003-listpcrs-fix-for-unsupported-disabled-alg-in-L.patch \
+DEPENDS = "tpm2-tss openssl curl autoconf-archive pkgconfig libgcrypt"
+SRC_URI = "git://github.com/01org/tpm2-tools.git;protocol=git;branch=master;nobranch=1;name=tpm2-tools;destsuffix=tpm2-tools \
```

Comment:
`branch=master` and `nobranch=1` contradict each other (`nobranch` seems to win ;)).
In this case you can use the tag for the release tied to the revision you use:
```sh
SRC_URI = "https://github.com/01org/tpm2-tools.git;protocol=git;branch=master;tag=3.1.3;name=tpm2-tools;destsuffix=tpm2-tools"
```

And since this is a fixed revision, you may as well rename the recipe `tpm2-tools_3.1.3.bb` and use `${PV}` directly for the tag:
```sh
SRC_URI = "https://github.com/01org/tpm2-tools.git;protocol=git;branch=master;tag=${PV};name=tpm2-tools;destsuffix=tpm2-tools"
```
... then remove `SRCREV`, `PVBASE` and `PV` statement.

Reply:
Since @eric-ch drew my attention here, you don't need `name` since there is only one git repo in SRC_URI.  Also, `destsuffix` can be dropped from `SRC_URI` and bitbake will extract to a directory named `git`.  Then below, you can change S to `S = "${WORKDIR}/git"` - that matches how git recipes are usually handled by OE.

Reply:
Addressed.